markdownlint.js

```diff
@@ -54,13 +57,15 @@ function readConfiguration(userConfigFile) {
   if (userConfigFile) {
     try {
       const userConfig = jsConfigFile
-        // Evaluate .js configuration file as code
-        ? require(path.resolve(processCwd, userConfigFile))
-        // Load JSON/YAML configuration as data
-        : markdownlint.readConfigSync(userConfigFile, configFileParsers);
+        ? // Evaluate .js configuration file as code
```

Comment:
This is unusual

Reply:
Agreed. I manually reformatted this in the latest commit series.